components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -87,9 +86,13 @@ export const WorkspaceEntry: FunctionComponent<Props> = ({ info, shortVersion })
                         </div>
                     </ItemField>
                     <ItemField className="w-2/12 flex my-auto">
-                        <Tooltip content={`Created ${dayjs(info.workspace.creationTime).fromNow()}`}>
+                        <Tooltip
+                            content={`Last Activate ${dayjs(
+                                info.status?.phase?.lastTransitionTime?.toDate(),
+                            ).fromNow()}`}
+                        >
                             <div className="text-sm w-full text-gray-400 overflow-ellipsis truncate">
-                                {dayjs(WorkspaceInfo.lastActiveISODate(info)).fromNow()}
+                                {dayjs(info.status?.phase?.lastTransitionTime?.toDate() ?? new Date()).fromNow()}
```

Comment:
` ?? new Date()` ? necessary? can be lastTransitionTime really be undefined?

Reply:
It's more safe than mark it with `!`

Reply:
I afraid we will pollute our code with it, but ok. Besides shapes our API should also have a contract and sometimes it should be fine to use `!` because we always probably things semantically.